README.md

```diff
@@ -1,5 +1,4 @@
-# swift-mongo-mobile
+# End of Life Notice
+MongoDB's swift-mongo-mobile is now officially end-of-life (EOL). No further development, bugfixes, enhancements, documentation changes or maintenance will be provided by this project and pull requests will no longer be accepted.
 
-## DISCLAIMER
-
-This driver is beta software. MongoDB SLAs do **NOT** apply to this beta software and it should **NOT** be used in production environments.
+Users are encouraged to migrate to the [MongoDB Realm](https://www.mongodb.com/realm) for mobile application development with MongoDB.
```

Comment:
Small nit - Is "the MongoDB Realm" correct? Should you remove the "the" or add "project" after MongoDB Realm?

Reply:
Ah thank you for catching, sloppy typo, I apologize! Removing "the"